membership-attribute-service/conf/routes

```diff
@@ -23,6 +23,7 @@ GET         /user-attributes/me/one-off-contributions
 
 PUT         /user-attributes/me/delivery-address/:contactId                     controllers.ContactController.updateDeliveryAddress(contactId: String)
 
+GET         /user-attributes/me/is-test-user                                    controllers.AttributeController.isTestUser
```

Comment:
we decided to go for a narrow scope of name rather than making it generic, as the function is unlikely to change in the short term.  But since manage will be the only client for now, it wouldn't be a problem to replace it later.